concepticondata/conceptlists/Castro-2015-608.tsv

```diff
@@ -367,22 +367,22 @@ Castro-2015-608-365	365	to gaze			望（远处）	to gaze into the distance
 Castro-2015-608-366	366	to face upward			仰（仰头的状态）	to face upward
 Castro-2015-608-367	367	to smell	2124	SMELL	闻（用鼻子）	to smell
 Castro-2015-608-368	368	to speak	1623	SPEAK	说（说汉话）	to speak, to say
-Castro-2015-608-369	369	to tell			说，指责（“我说你！”）	to reprimand, to advise
+Castro-2015-608-369	369	to tell	1711	TELL	说，指责（“我说你！”）	to reprimand, to advise
```

Comment:
I'd unmap this one. The translation "to reprimand, to advise" indicates a more specific type of telling.